packages/shared/src/contexts/payment/Paddle.tsx

```diff
@@ -230,13 +230,19 @@ export const PaddleSubProvider = ({
         items: [{ priceId, quantity: 1 }],
         customer: {
           email: user?.email,
+          ...(geo?.region && {
+            address: {
+              countryCode: geo?.region,
+            },
+          }),
```

Comment:
Btw. what happens if `geo?.region` is undefined, do they fallback to something or?

Reply:
User will need to fill this field, this is just a prefill but every field can be edited/added after by user ✔️

Reply:
Yes they do USD